TCC.Lib/Options/CompressionAlgo.cs

```diff
@@ -1,9 +1,29 @@
-﻿namespace TCC.Lib.Options
+﻿using System;
+
+namespace TCC.Lib.Options
 {
     public enum CompressionAlgo
     {
         Lz4,
         Brotli,
         Zstd
     }
+
+    [Flags]
+    public enum BenchmarkCompressionAlgo
+    {
+        Lz4 = 1 << 0,
+        Brotli = 1 << 1,
+        Zstd = 1 << 2,
+        All = Lz4 | Brotli | Zstd
+    }
+
+    [Flags]
+    public enum BenchmarkContent
+    {
+        UserDefined = 0,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename 'UserDefined' to 'None'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2346)